main.go

```diff
@@ -39,7 +39,11 @@ func main() {
 
 	e := echo.New()
 
-	r := echoswagger.New(e, "/v1", "doc", &echoswagger.Info{
+	var basePath string
+	if *appEnv != "" {
```

Comment:
I think it is not general usage.
`appEnv` is just environment variable.
How about get `basePath` from command argument or env?